.release-please-manifest.json

```diff
@@ -0,0 +1,3 @@
+{
```

Comment:
Is it indeed required to duplicate the version number? No way to extract it from cmake? Or inject it via tags?

Reply:
I wouldn't say duplicate. This version number is now leading, it's the one-and-only ground-truth. The version number in the CMakeLists.txt and, for example, the antora documentation will be updated as part of this action.

When the PR that this action generates is merged back to the main branch the tags and releases are created automatically.